feature-based-molecular-networking/tools/feature-based-molecular-networking/scripts/constants.py

```diff
@@ -0,0 +1,42 @@
+# constants for....
+
+# edges
+class EDGE:
+    TYPE_ATTRIBUTE = "EdgeType"
+    ION_TYPE = "MS1 annotation"  # ion identity molecular networking IIMN
+    COSINE_TYPE = "Cosine"  # molecular networking edge
+    # list of all types
+    TYPES = [ION_TYPE, COSINE_TYPE]
+
+    SCORE_ATTRIBUTE = "EdgeScore"  # score for all edge types (cosine, ion identity, ..)
+    NODE1_ATTRIBUTE = "node1"  #
+    NODE2_ATTRIBUTE = "node2"  #
+    MASS_DIFF_ATTRIBUTE = "mass_difference"  #
+    ANNOTATION_ATTRIBUTE = "EdgeAnnotation"  # edge annotation for all edge types
+    EXPLAINED_INTENSITY_ATTRIBUTE = "explained_intensity"  # explained intensity between the two MS2 spectra from MN
```

Comment:
Added constants.py to store and access node and edge attributes and values
import constants as CONST
CONST.EDGE.TYPE_ATTRIBUTE
CONST.EDGE.SCORE_ATTRIBUTE
CONST.NODE. ...